### DISCUSSION TITLE
Full part list

### DISCUSSION THREAD

**Discussion by Alameleon** [MiczFlor/RPi-Jukebox-RFID#2207] [Show and tell]
Hey hey,
I'm looking for a full list with all the recommended parts to build the phoniebox here on github.
I found many homepages with a full list, but I want it of the official MiczFlor Developer here. And yup, I'm new here on github and try to get an overview of this structure here.

**Comment by s-martin** [comment]
Hi, welcome!

There is not a „standard“ Phoniebox, the motto is [„everybody builds an original“](
![](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/docs/2024-Phoniebox-Calendar.jpg)
).

Nevertheless there are some recommendations, please check [here](https://github.com/MiczFlor/RPi-Jukebox-RFID#hardware-aka-shopping-list). The key feature of a Phoniebox is an RFID reader, the easiest ist probably the [Neuftech USB reader](https://www.amazon.de/Neuftech-Reader-Kartenlesegerät-Kartenleser-Kontaktlos/dp/B018OYOR3E).

Docs for version 2.x (will soon go in maintenance mode): https://github.com/MiczFlor/RPi-Jukebox-RFID#documentation

Docs for version 3.x (will be soon the default version): https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/README.md

Please check also the wiki: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki

Threads for inspiration: 

- https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1642
- https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/2115
- https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1331

You can also ask direct questions in our chat: https://matrix.to/#/#phoniebox_community:matrix.org


Happy building!

**Reply by Alameleon** [reply]
Hey thanks for your answer. I think, you missunderstood. I do not want to make a standard. But I never worked with RasPi and so I do not know which parts I cant (incompatible with Pi) use or how to braze it, what RFID-reader is compatible or what parts else I need.
Please don't get me wrong, but there are brilliant instructures with part lists in the internet. I'll use this to build an own phoniebox. But I want something official, too.

Hey hey, danke für deine Rückmeldung. Ich glaube, du weißt noch nicht ganz genau, was ich meine (an deinem "ist" habe ich es erkannt). Ich möchte keine standarisierte Phoniebox bauen, aber ich habe noch nie mit RasPi gearbeitet. Und darum weiß ich nicht, was für Teile ich nicht verwenden kann (Inkompatibilität) oder welche Stellen ich wie löten muss, welche RFID-Reader kompatibel sind oder was ich außer dem Pi und dem Reader außerdem noch benötige.
Versteh mich nicht falsch, es gibt eine Menge guter Anleitungen im Internet, die ich auch beim Bau verwenden werde. Allerdings möchte ich auch irgendwas offizilles haben.

**Reply by s-martin** [reply]
I hope the links I provided are a good start. The hardware recommendations are still valid.

Please ask any further questions you have here or in the chat.

You can also post what your Phoniebox should do (a Phoniebox for a 3 year old kid may be different than one for an adult) and the things you selected and you will get opinions here :)

**Comment by drstil** [comment]
I also want to build a Phoniebox but did not really find good information about needed raspberry. I have to build the box for a child, listening to onboard audio files, but also some online music streams and maybe some YouTube content. GPIO will be used for controlling. I will add a cheap USB sound card and the device will be used most time with a power bank (around 30.000 mAh), so one requirement is low power consumption. This power bank also supplies the 8 W stereo speakers and the amplifier. 
So I thought about a Raspberry Pi Zero 2 W (low power consumption), but was not really sure if this could lead to any performance issues, as a Pi 4 B is also recommended (https://github.com/MiczFlor/RPi-Jukebox-RFID#hardware-aka-shopping-list) and has much more CPU power. There are also no recommendations about needed RAM. 
If there could be performance issues (also about RAM as Zero 2 W only has 512 MB), it is no issue for myself to buy a more powerful raspberry, but I cannot decide by my one as I have no experience with that software.
If you would need other information to support my decision, I am happy to answer them! Thank you very much!

**Reply by s-martin** [reply]
Many people use the Zero 2W, because of its form factor.

keep in mind that the web interface may be slow, but depends if you want to actively use it besides setting up RFID cards. Startup time may also be longer on a slower Pi.

The current Spotify implementation for V2 uses Mopidy, which also needs more resources than the mpd. If you don’t plan to use Spotify install V2 classic or V3.

**Reply by pabera** [reply]
The goal for future3 is to work with lower performance Pi as much as possible. The goal is to save energy mainly.

The Zero 2 is the same hardware as a Pi 3B+, and both form factors work well for Phoniebox. Pi 4 should already be an overkill, most of the time the high performing CPU sits idle.

I am running a Zero 2 myself and it works great.
